funding/manager.go

```diff
@@ -1021,7 +1021,10 @@ func (f *Manager) stateStep(channel *channeldb.OpenChannel,
 
 		if !received {
 			// We haven't received FundingLocked, so we'll continue
-			// to the next iteration of the loop.
+			// to the next iteration of the loop after sleeping for
+			// one second.
+			time.Sleep(1 * time.Second)
```

Comment:
nit: could we use a `select` and to make use of `f.quit`?

Reply:
done!